H/hsakmt_roct/build_tarballs.jl

```diff
@@ -30,6 +34,7 @@ make install
 # ROCT-Thunk-Interface only supports Linux, seemingly only 64bit
 platforms = [
     Linux(:x86_64, libc=:glibc),
+    Linux(:x86_64, libc=:musl),
```

Comment:
In Yggdrasil we're now using Julia v1.6 which has new types for the platforms:
```suggestion
    Platform("x86_64", "linux"; libc=:glibc),
    Platform("x86_64", "linux"; libc=:musl),
```